haskell-hackage-stackage.nix

```diff
@@ -0,0 +1,38 @@
+{ pkgs }:
+let
+  # overriding logic so we can pass -I to nix, and overide the
+  # relevant import.
```

Comment:
I'm sure this makes sense to you, but it's quite mystifying to a reader.

Reply:
I've extended the comment to make this clearer.